geopandas/io/tests/test_file.py

```diff
@@ -607,6 +608,25 @@ def test_read_file_local_uri(file_path, engine):
     assert isinstance(gdf, geopandas.GeoDataFrame)
 
 
+def test_read_file_geojson_string_path(engine):
```

Comment:
If I'm looking correctly, this test is now actually being skipped in every CI env.

Reply:
The intent was to have it run in 3.11-dev, would have thought that would have worked.